modules/ROOT/pages/Development/Satisfactory/Audio.adoc

```diff
@@ -7,14 +7,215 @@ This page is a work in progress.
 
 == Wwise
 
-TODO
-
 Coffee Stain uses the Wwise audio system to play sounds in Satisfactory.
 
-At the time of writing it is very difficult for modders to interface with Wwise systems,
-so you should use Unreal's built in sound system instead.
+To create new audio events to play in your mod, you will need the Wwise project set up and integrated into the SML project.
+
+=== Setting Up Audio Buses
+
+By default when using Wwise audio only the `Master` volume slider will work.
+
+You will need to manually create the different Audio Buses that link to the other sliders.
+
+TODO: Waiting for CSS to provide the correct audio bus names.
+
+=== Importing Media Files to Wwise
+
+Open the Wwise project file, typically in `SMLProject/SMLProject_WwiseProject/SMLProject_WwiseProject.wproj`. Double-clicking this file will open Wwise.
+
+To make sure you are in the correct layout at the top, go to `Layout -> Designer`
+
+On the left-hand side, you will see the project explorer; select the Audio Tab.
+
+image:Satisfactory/Wwise/Wwise_Image1.png[Wwise_Image1, 350]
```

Comment:
Could you rename the image files to talk about what they are showcasing? Numbered file names is troublesome as inevitably another one will be added in the middle of the sequence

Reply:
Renamed images for better understanding.